src/styles/components/modal/styles.less

```diff
@@ -1,19 +1,13 @@
 & when (@modal-enabled) {
 
   .modal {
-    z-index: @z-index-modal-container;
```

Comment:
@jfurrow this looks right, right? I remember we made some effort to just have a single variable for the modal z-index. Maybe you can verify?